ffcx/codegeneration/integrals_generator.py

```diff
@@ -257,8 +238,6 @@ def generate_element_tables(self):
 
         tables = self.ir.unique_tables
         table_types = self.ir.unique_table_types
-        inline_tables = self.ir.integral_type == "cell" and not (self.ir.needs_rotations or self.ir.needs_reflections)
```

Comment:
This (and the same code on line 1095) is the only place that I used `needs_rotations` and `needs_reflections`, so these can now be removed from the ir (probably in a future PR not here?)